tests/e2e/test_incremental.py

```diff
@@ -118,7 +118,7 @@ def test_incremental():
     featurizer.apply(split=0, train=True, parallelism=PARALLEL)
     assert session.query(Feature).count() == len(train_cands[0])
     num_feature_keys = session.query(FeatureKey).count()
-    assert num_feature_keys == 526
+    assert num_feature_keys == 525
```

Comment:
See comment above.

Reply:
I broke the check into 4 (ie structural, visual, textual, tabular) and found that the difference came from "tabular".
You can see this at https://github.com/HiromuHota/fonduer/runs/809379470 (which is running as of writing).

Reply:
Yeah actually the first problem was that there was an `or` for the `is_tabular` while I changed that to `all`, now updated to `any`

But still there are some other difference
So this is what the old code shows that the new code doesn't
```
[
    "TAB_DIFF_TABLE_ROW_DIFF_[-1]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[1]",
    "TAB_DIFF_TABLE_COL_DIFF_[-1]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[4]",
    "TAB_DIFF_TABLE",
    "TAB_DIFF_TABLE_COL_DIFF_[0]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[0]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[2]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[5]",
    "TAB_DIFF_TABLE_MANHATTAN_DIST_[3]",
    "TAB_DIFF_TABLE_ROW_DIFF_[0]",
    "TAB_DIFF_TABLE_ROW_DIFF_[-2]",
    "TAB_DIFF_TABLE_ROW_DIFF_[-4]",
    "TAB_DIFF_TABLE_COL_DIFF_[1]",
    "TAB_DIFF_TABLE_ROW_DIFF_[1]"
]
```

And this is what the new code has, that the old code hasn't
```
[
    "TAB_WORD_DIFF_[1]",
    "TAB_CHAR_DIFF_[-6]",
    "TAB_CHAR_DIFF_[1]",
    "TAB_WORD_DIFF_[0]",
    "TAB_SAME_TABLE_MANHATTAN_DIST_[4]",
    "TAB_SAME_TABLE_ROW_DIFF_[-1]",
    "TAB_SAME_SENTENCE",
    "TAB_SAME_TABLE_ROW_DIFF_[1]",
    "TAB_SAME_TABLE_COL_DIFF_[1]",
    "TAB_CHAR_DIFF_[-7]",
    "TAB_SAME_TABLE_ROW_DIFF_[0]",
    "TAB_SAME_TABLE_MANHATTAN_DIST_[2]",
    "TAB_CHAR_DIFF_[0]",
    "TAB_SAME_TABLE_COL_DIFF_[-1]",
    "TAB_SAME_TABLE_ROW_DIFF_[-2]",
    "TAB_SAME_TABLE_ROW_DIFF_[-4]",
    "TAB_CHAR_DIFF_[-5]",
    "TAB_SAME_CELL",
    "TAB_SAME_TABLE_MANHATTAN_DIST_[0]",
    "TAB_SAME_TABLE_MANHATTAN_DIST_[1]",
    "TAB_WORD_DIFF_[-1]",
    "TAB_WORD_DIFF_[-2]",
    "TAB_SAME_TABLE_MANHATTAN_DIST_[5]"
]
```

I'm still trying to identify what is wrong there, but if are able to guide me faster there it would be helpful